content/c-sharp/concepts/Conditionals/conditionals.md

```diff
@@ -0,0 +1,152 @@
+
+After creating the variables we again start our `if statement`.
+
+`if` input1 is not equal to input 2 then set the variable of output to false. In our example, input1 was equal to input 2. The `if statement` check was false so it ran the code setting the value of output to false. 
+
+after that, we wrote a second `if statement` which checked if the output was equal to true. The output of this was false so it ran no code and wrote nothing to the console. if the first `if statement` would have been true. The variable output would get assigned to true and the second `if statement` would return true meaning the text *I returned true* would we have written in the console. 
+
+## else if statements
+
+An `else if statement` comes after an `if statement` if more comparing is needed and before an `else statement` we've added another variable to the mix.
+
+## Example 3
+```cs
+
+var input1 = 10;
+var input2 = 10;
+var input3 = 5;
+var output;
+
+if (input1 === input3){
+    output = true; // sets the value of output to true.
+} else if (input1 === input2){
+    output = true; // sets the value of output to true.
+} else {
+    output = false; // sets the value of output to false.
+}
+
+if(output === true){
+    Console.Writeline('I returned true')
+} else if (output === false) {
+    Console.Writeline('I returned false')
+} else {
+    return
+}
+
+```
+
+## Explanation of example 3
+
+We now start by creating four variables (`var`) and assigning a value to three of them. The fourth variable will be assigned in a later stage of the code.
+
+After creating the variables we again start our `if statement`.
+
+`if` input1 is equal to input3 then set the variable of output to true. In our example, input1 was not equal to input3. The `if statement` check was false so it ran the next part of the code.
+
+`else if` input1 is equal to input2 then set the variable of output to true. In our example, input1 was equal to input2 setting the value of output to true. 
+
+If this was also false the `else statement` would run setting the value of output to false.
+
+after that, we wrote a second `if statement` which checked if the variable output was equal to true. The output of this was true so it wrote *I returned true* to the console. if the first `if statement` would have been false the `else if statement` would run. This would check if output was equal to false. This would have been true and written *I returned false* to the console.
+
+## Ternary operator
+
+The conditional operator `?:` also known as the `ternary conditional operator` checks a boolean output and returns one of the two results depending on it being true or false. 
+
+## Example 4
+```cs
+
+string getInput1(input1) => input1 === 10 ? "I returned true" : "I returned false";
+
+Console.WriteLine(getInput1(10)); // Prints to the console "I returned true"
+Console.WriteLine(getInput1(5)); // Prints to the console "I returned false"
+
+```
+
+## Explanation of example 4
```

Comment:
I see 4 example, and while I do applaud the number of examples, I'd like to see either more meaning headings that show what conditional concept it's covering.